Bottom: 63e2129fa90d65f287b2c3ee706a130d1da7c34f
Top:    e6bff041d0c5413c6a16fc301b5d30d42c85fb00
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 22:44:07 +0000

Disable polygon heritage


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 950ea0d..885f268 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2848,8 +2848,10 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
   *res = NULL;
 
   /* Make copies for tracking polygon parentage (DEBUG) */
-  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
-      return err_no_memory;
+//  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
+//      return err_no_memory;
+  a_copy = NULL;
+  b_copy = NULL;
 
   /* Move the parentage information over onto the copy */
   if (a_copy != NULL)
